sections/ai_rules.tex

```diff
@@ -101,7 +103,7 @@ \subsection*{Combat against AI}
 \vfill
 
 \centering
-\includegraphics[width=0.9\linewidth]{\art/coin.png}
+\includegraphics[width=0.7\linewidth]{\art/coin.png}
 
```

Comment:
Don't resize, and please don't use different size for different languages. Just get rid of it.